backend/django/core/views/api_admin.py

```diff
@@ -312,10 +312,21 @@ def heat_map_data(request, project_pk):
     project = Project.objects.get(pk=project_pk)
 
     heatmap_data = irr_heatmap_data(project)
-    labels = list(
-        Label.objects.all().filter(project=project).values_list("name", flat=True)
-    )
-    labels.append("Skip")
```

Comment:
So I believe the point of this line was that for some IRR, one user might label it while the other skips it. This counts as a disagreement.